modules/delivery/delivery.service.js

```diff
@@ -0,0 +1,244 @@
+const request = require('request');
+const util = require('util')
+const post = util.promisify(request.post)
+const { 
+    CITY_NOT_FOUND,
+    WAREHOUSE_NOT_FOUND,
+ } = require('../../error-messages/delivery.message')
+
+class NovaPoshtaService {
+  async getNovaPoshtaCities(city){  
+      const res = await post(
+          process.env.NOVA_POSHTA_API_LINK,
+          { 
+            json: true,
+            body: {
+              modelName: 'Address',
+              calledMethod: 'getCities',
+              methodProperties: {
+                  FindByString: city
+              },
+              apiKey: process.env.NOVA_POSHTA_API_KEY
+            }
+          },
+      );
+
+      if(res.body.errors.lenght) {
```

Comment:
You have a typo here `lenght` instead of `length`

Reply:
fixed